common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -247,6 +253,7 @@ def get_html(self):
             'element_id': self.location.html_id(),
             'num_inputs': self.num_inputs,
             'submitted': self.submitted,
+            'js_includes': js_includes,
```

Comment:
Maybe?  Seems like that puts us in XModule-to-XBlock conversion territory, but I really don't know how much is XModule and how much is XBlock when it comes to implementing these abstract-ish methods.